all_examples_test.go

```diff
@@ -167,18 +167,43 @@ func ExampleTimeseriesEntry() {
 	// Content second blob value: data
 }
 
+func ExampleNode() {
+	handle, err := SetupHandle("qdb://127.0.0.1:30083", 120*time.Second)
```

Comment:
Could you extract a constant from `qdb://127.0.0.1:30083`, please? Something like `(un)secureClusterUri`.